client/finality-grandpa/src/lib.rs

```diff
@@ -1138,7 +1138,8 @@ fn local_authority_id(
 		Some(keystore) => voters
 			.iter()
 			.find(|(p, _)| {
-				SyncCryptoStore::has_keys(&**keystore, &[(p.to_raw_vec(), AuthorityId::ID)])
+				SyncCryptoStore::has_keys(&**keystore, &[(p.to_raw_vec(),
```

Comment:
This method is the perfect use case which should be optimized with the new `has_keys`.